test/runtime_tests.cc

```diff
@@ -763,6 +763,35 @@ namespace detail {
 		REQUIRE_FALSE(bm.has_active_buffers());
 	}
 
+	TEST_CASE("buffer_manager allows buffers deallocation", "[buffer_manager][dealloc]") {
+		celerity::distr_queue q;
+		buffer_id b_id;
+		auto& bm = runtime::get_instance().get_buffer_manager();
+		{
+			celerity::buffer<int, 1> b(celerity::range<1>(128));
+			b_id = celerity::detail::get_buffer_id(b);
+			q.submit([=](celerity::handler& cgh) {
+				celerity::accessor a{b, cgh, celerity::access::all(), celerity::write_only};
+				cgh.parallel_for<class UKN(i)>(b.get_range(), [=](celerity::item<1> it) {});
+			});
+			REQUIRE(bm.has_buffer(b_id));
+		}
+		celerity::buffer<int, 1> c(celerity::range<1>(128));
+		// 10 tasks so that 1st task horizon is arleady applied considering task_horizon_step_size=4
```

Comment:
Typo.

Also, should we set the task horizon step size in tests which depend on a specific value?